.github/workflows/size-limit.yml

```
@@ -5,26 +5,30 @@ on:
     branches:
       - main
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: null
```

actionlint (1.7.4)

"null" is invalid for permission for all the scopes. available values are "read-all" and "write-all"

(permissions)